tests/storage/storage_migration/utils.py

```diff
@@ -82,3 +87,14 @@ def verify_storage_migration_succeeded(
             file_content=CONTENT,
         )
         verify_vm_storage_class_updated(vm=vm, target_storage_class=target_storage_class)
+
+
+def get_storage_class_for_storage_migration(storage_class: str, cluster_storage_classes_names: list[str]) -> str | None:
```

Comment:
i think `None` is not needed here

Reply:
I'm not sure.. my ide wasn't happy with just str

Reply:
i think it just doesn't know how to treat `pytest.fail` :)

Reply:
i guess so :) changed